### PR TITLE
test/osd: Removed Commented Code - 2

### PR DESCRIPTION
**Pull request by joscollin**
Removed commented code found while reviewing another PR.

Fixes: http://tracker.ceph.com/issues/20207 (Partially)

Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins Retest this please

**Comment by joscollin** [comment]
Jenkins Retest this please

**Comment by joscollin** [comment]
Jenkins Retest this please

**Comment by joscollin** [comment]
Jenkins Retest this please

File: src/test/osd/types.cc [hunk 1]
```diff
@@ -703,7 +703,6 @@ TEST(pg_missing_t, claim)
 
 TEST(pg_missing_t, is_missing)
 {
-  // pg_missing_t::is_missing(const hobject_t& oid) const
```
Comment (liewegas) [inline src/test/osd/types.cc]: this is a real comment, not commented-out code

File: src/test/osd/types.cc [hunk 2]
```diff
@@ -712,7 +711,6 @@ TEST(pg_missing_t, is_missing)
     EXPECT_TRUE(missing.is_missing(oid));
   }
 
-  // bool pg_missing_t::is_missing(const hobject_t& oid, eversion_t v) const
```
Comment (liewegas) [inline src/test/osd/types.cc]: ditto

File: src/test/osd/types.cc [hunk 3]
```diff
@@ -970,7 +968,7 @@ TEST(pg_missing_t, add)
 
 TEST(pg_missing_t, rm)
 {
-  // void pg_missing_t::rm(const hobject_t& oid, eversion_t v)
+
```
Comment (liewegas) [inline src/test/osd/types.cc]: ...

**Comment by joscollin** [comment]
@liewegas Done.

**Comment by liewegas** [comment]
looks good.. can you squash back into 1 commit please?

**Comment by joscollin** [comment]
@liewegas squashed.

**Pull request merged**